content/documentation/pages/2-concepts/1-architecture.md

```diff
@@ -1,33 +1,330 @@
 ---
 path: 'concepts/architecture/'
 title: 'Architecture'
-description: 'Lorem markdownum madefacta, circumtulit aliis, restabat'
+description: "Introduction to Data Flow's Architecture."
 ---
 
 # Architecture
 
-TODO
+Thi guide explains the main concepts of Data Flow's architecture:
 
-## System Requirements
+- Data Flow's Server Components
+- The types of applications the server components can deploy for Streams and Batch Jobs
+- The microservice architecture of the deployed applications and how that is defined using a DSL
+- The platforms where they are deployed
 
-**Java:** Data Flow uses Java 8.
+Other sections of the guide explain
 
-**Database:** The Data Flow Server and Skipper Server need to have an RDBMS installed.
+- How the Server Components are secured and the tooling used to interact with them.
+- Runtime monitoring of Stream data pipelines.
```

Comment:
I think we stuck to "streaming data pipelines" wherever we used data pipelines - it'd be nice to use that for consistency